Automattic/newspack-blocks
# feat(i18n): add polish translations

### All Submissions:






### Changes proposed in this Pull Request:



Adds polish translations. 

### How to test the changes in this Pull Request:

1. Switch your site's language to polish, observe the Newspack Blocks strings are in polish

### Other information:
